tests/test_test_schema_section.py

```diff
@@ -78,21 +78,19 @@ def test_wrong_type():
 
 
 def test_min_length_violated():
-    # TODO: Make this pass (not implemented yet)
     """ Not adhering to minlength limitations should raise an error """
     with pytest.raises(
         DocumentationError,
-        match="Mismatched content. Response string violates the minimum string " "length defined in the schema",
+        match="The length of aa exceeds the minimum allowed length of 3",
```

Comment:
aa?

Reply:
It's just the printed value of the string passed as `data`:

```python
tester.test_schema_section(example_schema_string, "a" * 2)
```

Maybe we should add quotes to the print.